src/internal/observable/timer.ts

```diff
@@ -38,60 +38,62 @@ import { Subscriber } from '../Subscriber';
  * @see {@link index/interval}
  * @see {@link delay}
  *
- * @param {number|Date} [dueTime] The initial delay time to wait before
- * emitting the first value of `0`.
+ * @param {number|Date} [dueTime] The initial delay time specified as a Date object or as an integer denoting
+      milliseconds to wait before emitting the first value of 0`.
  * @param {number|SchedulerLike} [periodOrScheduler] The period of time between emissions of the
  * subsequent numbers.
  * @param {SchedulerLike} [scheduler=async] The {@link SchedulerLike} to use for scheduling
  * the emission of values, and providing a notion of "time".
  * @return {Observable} An Observable that emits a `0` after the
- * `initialDelay` and ever increasing numbers after each `period` of time
+ * `dueTime` and ever increasing numbers after each `period` of time
  * thereafter.
  * @static true
  * @name timer
  * @owner Observable
  */
-export function timer(dueTime: number | Date = 0,
-                      periodOrScheduler?: number | SchedulerLike,
-                      scheduler?: SchedulerLike): Observable<number> {
-  let period = -1;
-  if (isNumeric(periodOrScheduler)) {
-    period = Number(periodOrScheduler) < 1 && 1 || Number(periodOrScheduler);
-  } else if (isScheduler(periodOrScheduler)) {
-    scheduler = periodOrScheduler as any;
-  }
+export function timer(
+    dueTime: number | Date = 0,
+    periodOrScheduler?: number | SchedulerLike,
+    scheduler?: SchedulerLike
+): Observable<number> {
+    let period = -1;
+    if (isNumeric(periodOrScheduler)) {
+        period = (Number(periodOrScheduler) < 1 && 1) || Number(periodOrScheduler);
+    } else if (isScheduler(periodOrScheduler)) {
+        scheduler = periodOrScheduler as any;
+    }
 
-  if (!isScheduler(scheduler)) {
-    scheduler = async;
-  }
+    if (!isScheduler(scheduler)) {
+        scheduler = async;
+    }
```

Comment:
Hmm. In addition to deleting `"between emissions"` his last commit seems to have a whole bunch of whitespace/formatting changes. In particular, the indent size appears to have been changed from 2 spaces to 4, so whichever editor was used, it didn't pick up the `.editorconfig`.

Reply:
thanks for pointing, for some reasons VSCode is ignoring the editorconfig. I redo the changes and check it afterwards

Reply:
Support for `.editorconfig` isn't built-in. VS Code needs an extension added to see it supported. Asking, there are still a bunch of other whitespace changes and there are also parentheses added. That's not necessarily a problem, but if these are automated changes, I'd like to figure out why they are happening and prevent it from happening again - if only because it complicates reviews. Small, isolated changes would be preferable.